docs/data/joy/components/aspect-ratio/aspect-ratio.md

```diff
@@ -7,10 +7,11 @@ title: React Aspect Ratio component
 
 <p class="description">The Aspect Ratio component shapes the content with the specified ratio.</p>
 
-The default implementation combines `height: 0px` with percentage padding-bottom to create the proper aspect ratio of the content. (It will be replaced by the [CSS aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) in the future once it has sufficient various browser support).
+The default Aspect Ration implementation combines `height: 0px` with percentage padding-bottom to create the proper aspect ratio of the content.
+(It will be replaced by the [CSS aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) in the future once it has sufficient various browser support).
```

Comment:
```suggestion
(It will be replaced by the [CSS aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) in the future once it has sufficient browser support).
```